js/main.js

```diff
@@ -0,0 +1,189 @@
+              alt="nature card"
+              class="desktop-images"
+            />
+          </li>
+          <li>
+            <img
+              src="${element.imageMobile}"
+              alt="tarnner christensen from high school dropout to product designer at facebook"
+              class="mobile-images"
+            />
+          </li>
+        </ul>
+
+        <div class="tonic">
+          <h3>${element.title}</h3>
+          <div class="date">
+            <h3>${element.subTitle}</h3>
+                ${element.details
+                  .map(
+                    (
+                      x
+                    ) => ` <img src="img/Counter.png" alt="" class="counter" />
+            <p>${x}</p>`
+                  )
+                  .join('')}           
+          </div>
+          <p class="daily">
+           ${element.description}
+          </p>      
+          
+          <div>           
+            ${element.langs
+              .map((x) => `<button type="button">${x}</button>`)
+              .join('')}           
+          </div>
+          <button id='${element.id}' class="see-project">See Project</button>
+        </div>
+      </section>`;
+}
+
+document.querySelectorAll('.see-project').forEach((btn) => {
+  btn.addEventListener('click', () => {
+    const id = Number(btn.id);
+    const project = cardObject.find((p) => p.id === id);
+
+    document.querySelector('.pop').innerHTML = `
+      
+    <section class="modal-wrapper">
+      <div class="modal">
+        <div class="modal-title">
+          <h2>${project.title}</h2>
+          <div class='close-modal'>
+          <span class="iconify" data-icon="ep:close-bold"></span>
+          </div>
+        </div>
+        <div class="date">
+            <h3>${project.subTitle}</h3>
+                ${project.details
+                  .map(
+                    (
+                      x
+                    ) => ` <img src="img/Counter.png" alt="" class="counter" />
+            <p>${x}</p>`
+                  )
+                  .join('')}           
+          </div>
+        <img src="${project.imageDesktop}"/>
+        <div class="modal-content">
+          <p>
+            ${project.description}
+          </p>
+          <div>
+             <div>           
+            ${project.langs
+              .map((x) => `<button type="button">${x}</button>`)
+              .join('')}           
+          </div>
+            <div class="modal-btns">
+              <a href="${project.gitLink}" class="btn github-link">
+                See live
+                <span class="iconify iconbtn" data-icon="bytesize:github"></span>                                                                      
+              </a>
+              <a href="${project.srcLink}" class="btn source-link">
+                See source                
+                <span class="iconify iconbtnlink" data-icon="cil:external-link"></span>                            
+              </a>
```

Comment:
- [ ] I would recommend also kindly sticking to the design of these two buttons(height && width only).

The template | your design 
--------------|----------
![image](https://user-images.githubusercontent.com/65068771/146942574-bd8fdd2b-ca75-4f69-bf43-c7a970b44de5.png) | ![image](https://user-images.githubusercontent.com/65068771/146942915-ec749ba1-55b4-4fb3-9a21-9c8c33e2e62c.png)